rpcserver.go

```diff
@@ -286,10 +286,29 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 func (r *rpcServer) GetInfo(context.Context,
 	*taprpc.GetInfoRequest) (*taprpc.GetInfoResponse, error) {
 
+	// Retrieve the best block hash and height from the chain backend.
+	blockHash, blockHeight, err := r.cfg.Lnd.ChainKit.GetBestBlock(context.Background())
+	if err != nil {
+		// Handle error appropriately
```

Comment:
This comment doesn't give any additional info and should just be removed, here and below.